content/en/docs/concepts/services-networking/service.md

```diff
@@ -934,29 +931,74 @@ Service is a top-level resource in the Kubernetes REST API. More details about t
 API object can be found at:
 [Service API object](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#service-v1-core).
 
-## SCTP support
+## Supported protocols {#protocol-support}
+
+### TCP
+
+{{< feature-state for_k8s_version="v1.0" state="stable" >}}
+
+You can use TCP for any kind of service, and it's the default network protocol.
+
+### UDP
+
+{{< feature-state for_k8s_version="v1.0" state="stable" >}}
+
+You can use UDP for most services. For type=LoadBalancer services, UDP support
+depends on the cloud provider offering this facility.
+
+### HTTP
+
+{{< feature-state for_k8s_version="v1.1" state="stable" >}}
+
+If your cloud provider supports it, you can use a Service in LoadBalancer mode
+to set up external HTTP / HTTPS reverse proxying, forwarded to the Endpoints
+of the Service.
+
+{{< note >}}
+You can also use {{< glossary_tooltip term_id="ingress" >}} in place of Service
+to expose HTTP / HTTPS services.
+{{< /note >}}
+
+### PROXY protocol
+
+{{< feature-state for_k8s_version="v1.1" state="stable" >}}
+
+If your cloud provider supports it (eg, [AWS](https://kubernetes.io/docs/concepts/cluster-administration/cloud-providers/#aws)),
```

Comment:
nit: only cosmetic but i think you don't need extra space (> 1) after the comma and next word

Reply:
I can't see what I'd change to fix this, sorry.